docs/hpc/advanced/profiling/nsight_systems.md

```diff
@@ -0,0 +1,68 @@
+# Nsight Systems
+
+[NVIDIA Nsight Systems](https://developer.nvidia.com/nsight-systems) is a system-wide performance analysis tool designed to visualize an application’s algorithms, identify the largest opportunities to optimize, and tune to scale efficiently across any quantity or size of CPUs and GPUs, from large servers to our smallest systems-on-a-chip (SoCs).
+
+## Profile the code using nsight systems
+
+You can profile the code interactively or via a batch job.
+
+1. The first step is to load the appropriate modules.
+
+```bash
+module load tools/prod
+
+# Load the appropriate cuda module. We recommend using new versions as older versions are not supported on our systems for profiling.
```

Comment:
Worth specifing something like "versions 12.0 and newer"

Reply:
Corrected in 3d807fc507e7bdcfdb2dd978f9094d45d6b680c3 and 1b89b8fdac7d75afc158fe5655619fa0fd48357c